optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/SolverBenchmarkResult.java

```diff
@@ -337,7 +337,7 @@ protected void determineTotalsAndAverages() {
         if (!firstNonFailure) {
             int successCount = getSuccessCount();
             averageScore = totalScore.divide(successCount);
-            averageWorstScoreDifferencePercentage = totalWorstScoreDifferencePercentage.divide((double) successCount);
+            averageWorstScoreDifferencePercentage = totalWorstScoreDifferencePercentage.divide(successCount);
```

Comment:
Nitpick: I don't mind the explicit cast from int to double here. Autocasting for primitives has been shown to be dangerous. For example `double a = 9007199254740993L` becomes `9007199254740992.0`